src/playwright/testTypeImpl.ts

```diff
@@ -46,18 +48,27 @@ export async function runStepWithCustomLocation(
 }
 
 /**
- * Returns true if all fixtures of parent test found in child test.
+ * Returns true if this `test` function has all the fixtures we need
  */
-export function isParentChildTest(parent: TestTypeCommon, child: TestTypeCommon) {
-  if (parent === child) return false;
-  const childLocationsSet = new Set(
-    getTestFixtures(child).map((f) => locationToString(f.location)),
-  );
-  return getTestFixtures(parent).every((f) => {
-    return childLocationsSet.has(locationToString(f.location));
-  });
-}
+export function assertHasBddFixtures(test: TestTypeCommon) {
+  const allDefinedFixtures = Object.assign(
```

Comment:
Lets use `Set` here to avoid type assertion with `as Fixture<...>`:
```ts
  const allDefinedFixtures = new Set(
    getTestFixtures(test)
      .map(({ fixtures }) => Object.keys(fixtures || {}))
      .flat()
  );
```

Reply:
done